README.md

```diff
@@ -102,6 +102,16 @@ omnect-cli iot-hub-device-update import-update --help
 
 **Note**: The import process may take several minutes.
 
+### Remove update from IoT Hub
+This command removes an update from Azure Device Update for IoT Hub that was previously imported by `import-update` command.
+
+Detailed description:
+```sh
+omnect-cli iot-hub-device-update remove-update --help
+```
+
+**Note**: The import process may take several minutes.
```

Comment:
.... The remove process may take serveral minutes

Reply:
My tests didn't show any delays.

Reply:
done